### ISSUE TITLE
Enhancement - Can you add a flat shipping fee? 

### ISSUE THREAD

**Issue by carloswlkr** [geoffhumphrey/brewcompetitiononlineentry#992]
Version: 2.1.14

Installation URL: www.copapuravidaindie.com/copa2019

Is your installation hosted on brewcompetition.com or brewcomp.com? no

Description of Issue: We would like to charge a small, flat fee for each participant for shipping. This would be added to their total ($85 for 1st entry; $65 for second and subsequent entries; $15 flat shipping fee. Someone with 3 entries would pay $85+65+65+15. Is there any way to add this type of charge? 

Thanks!

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by geoffhumphrey** [comment]
There isn't via the software. There may be a way via PayPal, however.

I'm curious as to why you'd like to charge participants for shipping - are you assuming that responsibility for them? Sending them vouchers or ?? Admittedly, I'm a bit ignorant how professional competitions are run.

**Comment by carloswlkr** [comment]
We having the participants send or hadn deliver to centralized reception centers in each country, then we are arranging for the International shipping from those points to us in Costa Rica. This is so that we can handle the import permits ourselves. If each brewer shipped internationally to our contest, customs would hold up their beers in transit and require each a long list of import permits...

**Comment by geoffhumphrey** [comment]
Makes sense. Lots of logistics with international competitions, I'm gathering.

There is a way you could add the flat shipping fee. On or around line 58 in the /sections/pay.sec.php file, there's a line that looks like this:
```
$total_to_pay = $total_entry_fees - $total_paid_entry_fees;
```
On the next line, add this statement:
```
$total_to_pay = $total_to_pay + 15;
```
This will add the desired $15 flat shipping fee.

**Comment by carloswlkr** [comment]
Many thanks, Geoff! It looks like what we need is actually different from what I had originally posted. Turns out the shipping fee is per-entry on international entries, not per-participant. So in the example above, the total fees would be $85+15, +$65+15, +$65+15. If I simply increase the entry to $100 and $80 to include the shipping, we'll be over-charging local participants. One way I thought we could do this would be to include a check box whether the entry is local or international, and if the boolean is True, add the extra fee to each entry.  Any suggestions on this new spin?
